script.js

```diff
@@ -0,0 +1,74 @@
+
+// 카운트 하기 위해 할 일들 목록 각각 가져오기
+let waitingNum = document.getElementsByClassName("waiting");
+let completeNum = document.getElementsByClassName("compted");
```

Comment:
일관성 있는 클래스명을 사용해주시면 좋을 것 같아요!
그리고 `compted`같은 축약어는 되도록이면 지양하는게 좋아요! answer => ans, delete => del 정도는 괜찮지만 그렇게 많이 쓰이는(?) 형태는 아닌 것 같네요.

```suggestion
let waitingTodosCount = document.getElementsByClassName("waiting-todos-count");
let completedTodosCount = document.getElementsByClassName("completed-todos-count");
```

Reply:
compted 와 같은 것들도 지양하겠습니다 !